account_invoice_comment_template/migrations/13.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+field_renames = [
```

Comment:
This renaming is not preventing the m2m tables linking, so it doesn't serve. Even more, you don't want to rename fields, only columns (which is not the case), when renaming for preserving content in other place. The question here is: the m2m table has the same name? If that's the case, then you should rename the m2m table.